provider/kubernetes/kubernetes.go

```diff
@@ -199,10 +191,12 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					templateObjects.Frontends[r.Host+pa.Path] = &types.Frontend{
 						Backend:              r.Host + pa.Path,
 						PassHostHeader:       PassHostHeader,
+						PassTLSCert:          PassTLSCert,
 						Routes:               make(map[string]types.Route),
 						Priority:             priority,
 						BasicAuth:            basicAuthCreds,
 						WhitelistSourceRange: whitelistSourceRange,
+						EntryPoints:          EntryPoints,
```

Comment:
Are the entry points needed for the pass TLS functionality? If not, then I'd prefer to handle them in a separate PR.

Reply:
Did i really need to make two separate PR?